### PR TITLE
Make copilot postgres ready

### PR DESCRIPTION
**Pull request by talvasconcelos**
Various fixes for database compatibility and a few minor HTML fixes

### CODE REVIEW

File: lnbits/extensions/copilot/crud.py [hunk 1]
```diff
@@ -55,15 +55,14 @@ async def create_copilot(
             show_message,
             show_ack,
             show_price,
-            lnurl_title,
```
Comment (fiatjaf) [inline lnbits/extensions/copilot/crud.py:58]: Why did you remove this column?

**Comment by talvasconcelos** [comment]
it was a duplicate!
